src/styles.css

```diff
@@ -1,3 +1,117 @@
+/* @settings
+
+name: Habit Tracker 21
+id: ht21
+settings:
+  -
+    id: ht21-cell
+    title: Table cell font
+    type: heading
+    level: 3
+  -
+    id: ht21-cell-fontfamily
+    title: Header font family
+    type: variable-text
+    description: Used for the header row
+    default: '"Source Code Pro", monospace'
+  -
+    id: ht21-cell-font-size
+    title: Header font size
+    type: variable-number
+    description: Used for the header row
+    default: 13
+    format: px
+  -
+    id: ht21-line
+    title: Line width and color
+    type: heading
+    level: 3
+  -
+    id: ht21-border-width
+    title: Line width
+    description: Width of line between rows
+    type: variable-number
+    default: 1
+    format: px
+  -
+    id: ht21-border-color
+    title: Line color
+    description: Color of line between rows
+    type: variable-color
+    format: hex
+    opacity: false
+    default: '#363636'
+  -
+    id: ht21-tick
+    title: Tick bar
+    type: heading
+    level: 3
+  -
+    id: ht21-tick-height
+    title: Height of tick element
+    type: variable-number
+    default: 16
+    format: px
+  -
+    id: ht21-tick-width
+    title: Height of tick element
+    type: variable-number
+    default: 20
+    format: px
+  -
+    id: ht21-tick-radius
+    title: Radius of tick endings
+    type: variable-number
+    default: 10
+    format: px
+  -
+    id: ht21-tick-color
+    title: Color
+    type: variable-color
+    opacity: false
+    format: hex
+    default: '#8a5cf5'
+  -
+    id: ht21-color
+    title: Background color
+    type: heading
+    level: 3
+  -
+    id: ht21-tick-hover-color
+    title: Background hover color on tick
+    type: variable-color
+    opacity: false
+    format: hex
+    default: '#363636'
+  -
+    id: ht21-row-hover-color
+    title: Background hover color on row
+    type: variable-color
+    opacity: false
+    format: hex
+    default: '#242424'
+  -
+    id: ht21-weekend-color
+    title: Background color on weekend columns
+    type: variable-color
+    opacity: false
+    format: hex
+    default: '#242424'
+*/
+:root {
+  --ht21-cell-fontfamily: "Source Code Pro", monospace;
+	--ht21-cell-font-size: 13px;
+	--ht21-border-width: 1px;
+	--ht21-border-color: #363636;
+	--ht21-tick-height: 16px;
+	--ht21-tick-width: 20px;
+	--ht21-tick-color: #8a5cf5;
```

Comment:
This should be assigned `var(--checkbox-color)` otherwise it will break HT24 in themes other than the default obsidian theme. the same applies to all the cases where a variable was used before: --ht21-border-width, --ht21-row-hover-color. Will that still work with the plugin you're trying to support?

Reply:
I'm not sure how to use a variable in this type of setting.  Let me ponder a little on that, and feel free to suggest a solution if you see one. After all this is your project,  and I just wanted to chip in on how to make it even better and more customisable.